release/device-sdk-go.yaml

```diff
@@ -1,10 +1,14 @@
 ---
 name: 'device-sdk-go'
-version: '2.2.0'
-releaseName: 'kamakura'
+version: '2.3.0'
+releaseName: 'levski'
 releaseStream: 'main'
 repo: 'https://github.com/edgexfoundry/device-sdk-go.git'
-commitId: '4f64ab83298cb604fe88218673904389f4010661'
+commitId: 'b76771b07adf0b8c2e38dd8a749adeac8b55b549'
 gitTag: true
 dockerImages: false
-gitHubRelease: false
\ No newline at end of file
+gitHubRelease: false
+apiInfo: [
+   nextReleaseVersion: '2.4.0',
+   reviewers: edgexfoundry/edgex-committers
```

Comment:
The edgex-committers team seems a bit out of date. It contains a few members who are no longer active in the project. I suggest: `edgexfoundry/device-sdk-go-committers`

Reply:
All our committer lists need cleaning up. :-(